kong/pdk/log.lua

```diff
@@ -909,14 +918,14 @@ local function new_log(namespace, format)
     if not buf then
       error(err, 2)
     end
-
+    
```

Comment:
Do we need those whitespaces? 🤔 (i mean tabs)

Reply:
no, not sure where they come from :) removed.